Added OEPs and fixed ordering

WyattMufson opened the pull request.

qiluge commented:
hello, OEP-9 https://github.com/ontio/OEPs/pull/50 has been already used, please update your oep index.

WyattMufson commented:
@qiluge I switched it so that it now shows the current OEP9 proposal. I added OEP15 for PR #56. I also moved OEPs 6 and 7 to be in the right place (and added the author to OEP7). @javajoker @Honglei-Cong

WyattMufson commented:
@javajoker any comments on this?
